cmd/test-image/tcp/tcp.go

```diff
@@ -0,0 +1,46 @@
+package tcp
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/docker/libnetwork/cmd/test-image/server"
+	"github.com/sirupsen/logrus"
+)
+
+var basePaths = map[string]server.HTTPHandlerFunc{
+	"/tcp_ping": tcp_ping,
+	"/tcp_pong": tcp_pong,
+}
+
+func Init(s *server.TestServer) {
+	s.RegisterHandler(basePaths)
+}
+
+func tcp_ping(s *server.TestServer, w http.ResponseWriter, r *http.Request) {
+	logrus.Info("tcp_ping")
+	r.ParseForm()
+
+	logrus.Infof("form: %v", r.Form)
+	// toPingList := strings.Split(r.Form, sep)
+
+	for _, v := range r.Form {
+		for _, target := range strings.Split(v[0], ",") {
+			path := fmt.Sprintf("http://%s/tcp_pong", target)
+			_, err := http.Get(path)
```

Comment:
We probably eventually want to add support for timeouts here (probably as a parameter of the query).  But probably not needed for a first shot.

Reply:
yep makes sense